cmake/Modules/FindRADAREMUL.cmake

```diff
@@ -30,7 +30,7 @@ if( NOT RADAREMUL_LIBRARY ) # didn't find the library, so build it from source
     endif()
     set( radaremul "radaremul")
     set( BUILD_RADAREMUL "ON" CACHE INTERNAL "Build radaremul library" )
-    add_subdirectory(${CMAKE_SOURCE_DIR}/libsrc/radaremul)
+    #add_subdirectory(${CMAKE_SOURCE_DIR}/src/caps/radaremul)
```

Comment:
Why to comment out this line?

Reply:
Hi, If I leave it as unblocked, it failed when doing cmake test. I think this seems to occur since this is already declared in  CMakeLists.txt at root directory.